packages/models/proto/v2/authorization/key.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package authorization.v2;
+
+message KeyEntryV2 {
+  string keyId = 1;
+  KeyV2 value = 2;
+}
+
+message KeyV2 {
+  optional string relationshipId = 1;
+  string kid = 2;
+  string encodedPem = 3;
+  KeyUseV2 use = 4;
+  string algorithm = 5;
+  string createdAt = 6;
+  string name = 7;
+  optional string userId = 8;
+}
+
+enum KeyUseV2 {
+  SIG = 0;
+  ENC = 1;
+}
```

Comment:
Just for curiosity: why this starts from "0"?

Reply:
As far as I understand, `proto3` requires enum entries to start with `0`.

Reply:
Okkay, this is an enum, didn't notice